design/operator-architecture.asciidoc

```diff
@@ -0,0 +1,89 @@
+# Keycloak Operator High Level Architecture
+
+* **Status**: Draft #1
+* **JIRA**: https://issues.redhat.com/browse/KEYCLOAK-10319[KEYCLOAK-10319]
+* **Source of the images**: https://docs.google.com/presentation/d/13N5ClXXXcxKjXgor72rUdtMBN07_GneUkhTTwBsy_tE/edit?usp=sharing[Google Slides]
+
+This document shows a big picture how all the components and layers of Keycloak Operator work together. The blueprint presented here shall be used a guideline for designing features as well as integrating community Pull Requests.
+
+## Manifesto
+
+We envision Keycloak Operator as:
+
+* an opinionated way of installing and maintaining Keycloak installation on both Kubernetes and OpenShift.
+* an example of a recommended way of installing, configuring, managing custom extension and themes.
+* the thinnest possible layer on top of Keycloak Container and Keycloak Server.
+
+## Introduction
+
+Keycloak Operator was re-designed from the scratch by a small team withing RHMI and RHSSO groups. The implementation plan was described in the [Initial Operator Design Document](operator.md). At the end of a 3 sprint cycle, the team delivered a Level 4 Operator, that was pushed into the [Operatorhub](https://operatorhub.io/operator/keycloak-operator). Once the initial implementation was done, the Operator was handed over to the Keycloak/RHSSO Team.
```

Comment:
The [Initial Operator Design Document](operator.md) and [Operatorhub](https://operatorhub.io/operator/keycloak-operator) links are not working here. I don't believe they are formatted for asciidoc correctly

Reply:
Thanks for spotting this. Fixed.